helpers/commit_helpers.rb

```diff
@@ -1,10 +1,14 @@
 module CommitHelpers
   def commit_url(current_page)
+    return current_page.data.commit_url if current_page.data.commit_url
+
     commit_sha = `git log -1 --format=oneline #{source_file(current_page)} | cut -d ' ' -f 1`.strip
     "https://github.com/alphagov/govuk-developer-docs/commit/#{commit_sha}"
   end
 
   def last_updated(current_page)
+    return current_page.data.last_committed if current_page.data.last_committed
```

Comment:
It comes from ProxyPages in the first commit, where it's set like `last_committed: page[:latest_commit][:timestamp]`. I suppose I could set it as `current_page.data.latest_commit.timestamp` to keep it consistent?

Reply:
Ahh, I see: I missed that below the main change in the fetcher class. I got confused because of the similar naming in that class (`last_commit`), so assumed it was coming from there. It's quite a minor point, but I would have found it easier to follow if we just passed the whole hash through as `last_commit`.

Reply:
I've refactored this out now, we just pass a hash around instead.